{
  "done": [
    "pyproj",
    "libspatialite",
    "vtk",
    "visan",
    "paraview",
    "geotiff",
    "gdal",
    "qgis",
    "pygplates",
    "postgis",
    "mapserver"
  ],
  "in-pr": [
    "magics",
    "r-proj4",
    "magics-metview-batch",
    "rasterio",
    "fimex",
    "magics-metview"
  ],
  "awaiting-pr": [],
  "not-solvable": [
    "r-terra",
    "r-sf",
    "r-rgdal",
    "r-gdalcubes",
    "pynio",
    "ncl"
  ],
  "awaiting-parents": [
    "cdo",
    "r-gribr",
    "r-lwgeom"
  ],
  "bot-error": [],
  "_feedstock_status": {
    "cdo": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "gdal": {
      "num_descendants": 12,
      "immediate_children": [
        "mapserver",
        "ncl",
        "postgis",
        "pygplates",
        "pynio",
        "qgis",
        "r-gdalcubes",
        "r-rgdal",
        "r-sf",
        "r-terra",
        "rasterio"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/gdal-feedstock/pull/744",
      "pr_status": "unknown"
    },
    "geotiff": {
      "num_descendants": 13,
      "immediate_children": [
        "gdal",
        "qgis"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/geotiff-feedstock/pull/54",
      "pr_status": "unknown"
    },
    "libspatialite": {
      "num_descendants": 13,
      "immediate_children": [
        "gdal",
        "qgis"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/libspatialite-feedstock/pull/93",
      "pr_status": "unknown"
    },
    "magics": {
      "num_descendants": 1,
      "immediate_children": [
        "cdo"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/magics-feedstock/pull/121",
      "pr_status": "unstable"
    },
    "ncl": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "not solvable (<a href=\"https://github.com/regro/cf-scripts/actions/runs/4832555858\">bot CI job</a>): main: ['linux_64_: Encountered problems while solving:\\n  - package libgdal-3.6.3-h0e7c1b1_3 requires libjpeg-turbo >=2.1.5.1,<3.0a0, but none of the providers can be installed\\n', 'osx_64_: Encountered problems while solving:\\n  - package libgdal-3.6.3-h32fa94e_6 requires libjpeg-turbo >=2.1.5.1,<3.0a0, but none of the providers can be installed\\n', 'osx_arm64_: Encountered problems while solving:\\n  - package libgdal-3.6.3-h0a7289d_7 requires libjpeg-turbo >=2.1.5.1,<3.0a0, but none of the providers can be installed\\n']"
    },
    "paraview": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/paraview-feedstock/pull/181",
      "pr_status": "unknown"
    },
    "postgis": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/postgis-feedstock/pull/93",
      "pr_status": "unknown"
    },
    "pynio": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "not solvable (<a href=\"https://github.com/regro/cf-scripts/actions/runs/4832555858\">bot CI job</a>): main: ['linux_64_numpy1.21python3.10.____cpython: Encountered problems while solving:\\n  - nothing provides freeglut >=2.8.1,<3.0a0 needed by jasper-2.0.14-h07fcdf6_0\\n', 'linux_64_numpy1.21python3.8.____cpython: Encountered problems while solving:\\n  - nothing provides freeglut >=2.8.1,<3.0a0 needed by jasper-2.0.14-h07fcdf6_0\\n', 'linux_64_numpy1.21python3.9.____cpython: Encountered problems while solving:\\n  - nothing provides freeglut >=2.8.1,<3.0a0 needed by jasper-2.0.14-h07fcdf6_0\\n', 'linux_64_numpy1.23python3.11.____cpython: Encountered problems while solving:\\n  - nothing provides freeglut >=2.8.1,<3.0a0 needed by jasper-2.0.14-h07fcdf6_0\\n', 'osx_64_numpy1.21python3.10.____cpython: Encountered problems while solving:\\n  - package jasper-2.0.14-h0129ec2_2 requires jpeg, but none of the providers can be installed\\n', 'osx_64_numpy1.21python3.8.____cpython: Encountered problems while solving:\\n  - package jasper-2.0.14-h0129ec2_2 requires jpeg, but none of the providers can be installed\\n', 'osx_64_numpy1.21python3.9.____cpython: Encountered problems while solving:\\n  - package jasper-2.0.14-h0129ec2_2 requires jpeg, but none of the providers can be installed\\n', 'osx_64_numpy1.23python3.11.____cpython: Encountered problems while solving:\\n  - package jasper-2.0.14-h0129ec2_2 requires jpeg, but none of the providers can be installed\\n']"
    },
    "pyproj": {
      "num_descendants": 1,
      "immediate_children": [
        "qgis"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/pyproj-feedstock/pull/135",
      "pr_status": "unknown"
    },
    "qgis": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/qgis-feedstock/pull/319",
      "pr_status": "unknown"
    },
    "r-proj4": {
      "num_descendants": 1,
      "immediate_children": [
        "r-gribr"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/r-proj4-feedstock/pull/40",
      "pr_status": "unstable"
    },
    "r-rgdal": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "not solvable (<a href=\"https://github.com/regro/cf-scripts/actions/runs/4830267965\">bot CI job</a>): main: ['linux_64_r_base4.1: Encountered problems while solving:\\n  - package r-base-4.1.0-h9e01966_0 requires pcre2 >=10.36,<10.37.0a0, but none of the providers can be installed\\n', 'osx_64_r_base4.1: Encountered problems while solving:\\n  - package r-base-4.1.0-h4d23a9d_0 requires pcre2 >=10.36,<10.37.0a0, but none of the providers can be installed\\n']"
    },
    "r-sf": {
      "num_descendants": 1,
      "immediate_children": [
        "r-lwgeom"
      ],
      "pre_pr_migrator_status": "not solvable (<a href=\"https://github.com/regro/cf-scripts/actions/runs/4832555858\">bot CI job</a>): main: ['linux_64_r_base4.1: Encountered problems while solving:\\n  - package r-base-4.1.0-h9e01966_0 requires pcre2 >=10.36,<10.37.0a0, but none of the providers can be installed\\n', 'linux_aarch64_r_base4.1: Encountered problems while solving:\\n  - package r-base-4.1.0-hfd28f70_0 requires pcre2 >=10.36,<10.37.0a0, but none of the providers can be installed\\n', 'linux_ppc64le_r_base4.1: Encountered problems while solving:\\n  - package r-base-4.1.0-h0f82cec_0 requires pcre2 >=10.36,<10.37.0a0, but none of the providers can be installed\\n', 'osx_64_r_base4.1: Encountered problems while solving:\\n  - package r-base-4.1.0-h4d23a9d_0 requires pcre2 >=10.36,<10.37.0a0, but none of the providers can be installed\\n']"
    },
    "rasterio": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/rasterio-feedstock/pull/264",
      "pr_status": "unstable"
    },
    "vtk": {
      "num_descendants": 1,
      "immediate_children": [
        "visan"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/vtk-feedstock/pull/293",
      "pr_status": "unknown"
    },
    "r-gribr": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "r-lwgeom": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "magics-metview": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/magics-metview-feedstock/pull/64",
      "pr_status": "unstable"
    },
    "magics-metview-batch": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/magics-metview-batch-feedstock/pull/55",
      "pr_status": "unstable"
    },
    "fimex": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/fimex-feedstock/pull/55",
      "pr_status": "unstable"
    },
    "r-gdalcubes": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "not solvable (<a href=\"https://github.com/regro/cf-scripts/actions/runs/4831403240\">bot CI job</a>): main: ['linux_64_r_base4.1: Encountered problems while solving:\\n  - package libgdal-3.6.3-h0e7c1b1_3 requires libnetcdf >=4.9.1,<4.9.2.0a0, but none of the providers can be installed\\n', 'linux_64_r_base4.2: Encountered problems while solving:\\n  - package r-base-4.2.0-h1ae530e_0 requires icu >=58.2,<59.0a0, but none of the providers can be installed\\n', 'osx_64_r_base4.1: Encountered problems while solving:\\n  - package libgdal-3.6.3-h500a207_2 requires libnetcdf >=4.9.1,<4.9.2.0a0, but none of the providers can be installed\\n', 'osx_64_r_base4.2: Encountered problems while solving:\\n  - package r-base-4.2.0-h85fa70c_0 requires icu >=70.1,<71.0a0, but none of the providers can be installed\\n']"
    },
    "r-terra": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "not solvable (<a href=\"https://github.com/regro/cf-scripts/actions/runs/4832555858\">bot CI job</a>): main: ['linux_64_r_base4.1: Encountered problems while solving:\\n  - package libgdal-3.6.3-h0e7c1b1_3 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n', 'linux_64_r_base4.2: Encountered problems while solving:\\n  - package libgdal-3.6.3-h0e7c1b1_3 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n', 'linux_aarch64_r_base4.1: Encountered problems while solving:\\n  - package libgdal-3.6.3-h09f3c6d_3 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n', 'linux_aarch64_r_base4.2: Encountered problems while solving:\\n  - package libgdal-3.6.3-h09f3c6d_3 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n', 'linux_ppc64le_r_base4.1: Encountered problems while solving:\\n  - package libgdal-3.6.3-h1089159_8 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n', 'linux_ppc64le_r_base4.2: Encountered problems while solving:\\n  - package libgdal-3.6.3-h1089159_8 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n', 'osx_64_r_base4.1: Encountered problems while solving:\\n  - package libgdal-3.6.3-h32fa94e_6 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n', 'osx_64_r_base4.2: Encountered problems while solving:\\n  - package libgdal-3.6.3-h32fa94e_6 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n']"
    },
    "visan": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/visan-feedstock/pull/15",
      "pr_status": "unknown"
    },
    "mapserver": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/mapserver-feedstock/pull/25",
      "pr_status": "unknown"
    },
    "pygplates": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/pygplates-feedstock/pull/5",
      "pr_status": "unknown"
    }
  }
}